### PR TITLE
Data generation script

### PR DESCRIPTION
**Pull request by mtkane0**
Completed script that generates however many records needed

- Generates fake values using Faker

- Stores all records in JSON format, placed in db/products.json

### CODE REVIEW

File: db/data.js [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+  let fakePhotos = '';
+  for (let i = 0; i < numOfPhotos; i += 1) {
+    const thumbnail = faker.image.imageUrl(300, 400);
+    const url = faker.image.imageUrl(650, 800);
+    fakePhotos
+      += `{
+            "thumbnail_url": "${thumbnail}",
+            "url": "${url}"
+          }`;
+    fakePhotos += (i === numOfPhotos - 1) ? '' : ',';
```
Comment (utkarshsingh77) [inline db/data.js:40]: nice

File: db/data.js [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+             "value": "${value}"
+            }`;
+    fakeFeatures += (i === numOfFeatures - 1) ? '' : ',';
+  }
+  return fakeFeatures;
+};
+
+const createPhotos = () => {
+  const MAX = 10;
+  const MIN = 6;
```
Comment (utkarshsingh77) [inline db/data.js:29]: I appreciate your making variables for this instead of just putting the numbers in the formula

File: package.json [hunk 3]
```diff
@@ -29,6 +29,8 @@
     "axios": "^0.19.2",
     "compression": "^1.7.4",
     "express": "^4.17.1",
+    "faker": "^5.1.0",
+    "fs": "0.0.1-security",
```
Comment (utkarshsingh77) [inline package.json:33]: this is native

**Review by utkarshsingh77 — APPROVED** [review]
Overall, very good job. Minor comments made.

**Comment by mtkane0** [comment]
Thanks, Karsh. Great teamwork.

**Pull request merged**